### PR TITLE
DevKit updates for 2.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
any idea why this is failing ? @sonata-project/contributors 
Should we skip the test since this repository is not maintained ?

**Comment by dbu** [comment]
seems unrelated to the PR, so the test is broken in 2.x as well. lets mark the test as incomplete at the place where it breaks, with your reason.

**Review by VincentLanglet — DISMISSED** [review]

**Review by VincentLanglet — APPROVED** [review]

File: phpunit.xml.dist [hunk 1]
```diff
@@ -17,17 +14,18 @@
...
+        </include>
+    </coverage>
 
     <php>
         <ini name="precision" value="8" />
         <env name="KERNEL_CLASS" value="Sonata\DoctrinePHPCRAdminBundle\Tests\Fixtures\App\Kernel" />
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
-        <env name="SYMFONY_PHPUNIT_VERSION" value="8.5" />
+        <!-- Use max[self]=0 when https://github.com/symfony/symfony/issues/40400 is fixed -->
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=99999" />
```
Comment (dbu) [inline phpunit.xml.dist:27]: good point - for this repo which is legacy!

**Review by dbu — COMMENTED** [review]
thanks

**Pull request merged**